apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/journalpost/EndreAvsenderMottaker.tsx

```diff
@@ -64,14 +65,20 @@ export const EndreAvsenderMottaker = ({
           />
 
           <br />
+          <SoekPersonPdl open={open} setOpen={setOpen} oppdaterBruker={lagreEndretMottaker} />
 
           <HStack gap="4" justify="end">
             <Button variant="tertiary" onClick={avbryt} size="small">
               Avbryt
             </Button>
-            <Button variant="secondary" onClick={handleSubmit(lagreEndretMottaker)} size="small">
+            <Button
+              variant="secondary"
+              onClick={handleSubmit((avsendermottaker) => lagreEndretMottaker(avsendermottaker))}
+              size="small"
+            >
               Lagre
             </Button>
+            <Button onClick={() => setOpen(!open)}>Avansert Søk</Button>
```

Comment:
Hvorfor ikke putte knappen i modal-komponentet? Så slipper du så rotete kode

Reply:
Hva mener duher ,denne er for å få åpnet modalen.